bedrock/security/templates/security/web-bug-bounty.html

```diff
@@ -12,99 +12,140 @@ <h1 itemprop="name" class="title-shadow-box">Web and Services Bug Bounty Program
+            <td>$2500</td>
+            <td>$1000</td>
+            <td>ACK</td>
+          </tr>
+          <tr>
+            <td>XSS<sup>4</sup> (blocked by CSP)</td>
+            <td>$500</td>
+            <td>ACK</td>
+            <td>--</td>
+          </tr>
+          <tr>
+            <td>Clickjacking<sup>5</sup></td>
+            <td>$500</td>
+            <td>$250</td>
+            <td>--</td>
+          </tr>
+          <tr>
+            <td>Open Redirects</td>
+            <td>ACK</td>
+            <td>ACK</td>
+            <td>ACK</td>
+          </tr>
+        </tbody>
+      </table>
 
       <ol>
-        <li>What is the necessary position of the attacker? (Example: Logged in to x website, unauthenticated, etc.)</li>
-        <li>Are there any other assumptions that must be made about the victim? (Example: only affects specific web browser versions, requires that the user can be tricked into clicking on a link, etc. </li>
-        <li>How might a malicious actor use this attack? (Example: Steal victim’s cookies, redirect to malicious website, read files on server outside web root, remote OS command injection, XSS users, etc.)</li>
+        <li>Excludes community websites</li>
+        <li>Acknowledgement on our 
+          <a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">hall of fame</a>
+        </li>
+        <li>Significant actions only, such as changing email/passwords, deleting accounts, etc.</li>
+        <li>Excludes self-XSS</li>
+        <li>Lack of clickjacking protection (XFO, CSP) is insufficient to claim a bounty</li>
       </ol>
 
-      <h3>What is the step-by-step exploit process?</h3>
-
-      <p>Please describe the multi-step process required, step by step, in the right order. If there are multiple parties involved, please describe them all clearly. For example:</p>
-
-      <blockquote>
-        <ol>
-          <li>Visit the url https://xyz.firefox.com/somepage/form?name=value&amp;name2=value2</li>
-          <li>Log in</li>
-          <li>Click the “Frobozz” button</li>
-          <li>Place order for “Anti-Grue Cream” to shipping location “small mailbox,” continue navigation to the Shipping Address page.</li>
-          <li>Add the following payload <code>"&gt;&lt;img src=x onerror=alert(document.domain);// &gt;</code> into the “City” field, then click Submit.</li>
-          <li>The XSS payload ends up in the POST parameter name d oddly_angled_room</li>
-          <li>Visitors to the page /puzzles are affected by stored XSS</li>
-        </ol>
-      </blockquote>
 
-      <p>Include the HTTP request(s) including any necessary header values and POST parameters with your bug submission.</p>
-
-      <p>Please avoid videos. They take time to create and much more time for us to interpret.</p>
-
-      <p>Repeat the attack using only your own description in order to prevent errors and omissions, update documentation.</p>
-
-      <h3>XSS reporting tips</h3>
+      <h2>Exclusions</h2>
```

Comment:
It'd be nice to have an anchor here to email as reference for errant bounty reports (future..)